### PR TITLE
[CS] Standardize docblock asterisks

### PR DESCRIPTION
**Pull request by Slamdunk**
The fix was applied with [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) rule `phpdoc_opening_closing`

### CODE REVIEW

**Review by Majkl578 — APPROVED** [review]
LGTM, ready to merge once we are done rebasing & merging develop into master.

**Comment by Slamdunk** [comment]
Rebased

**Comment by Ocramius** [comment]
LGTM :+1:

**Pull request merged**
